Formula/c/curl.rb

```diff
@@ -80,7 +80,10 @@ def install
     ]
 
     args << if OS.mac?
-      "--with-gssapi"
+      %w[
+        --with-apple-sectrust
+        --with-gssapi
+      ]
     else
       "--with-gssapi=#{Formula["krb5"].opt_prefix}"
     end
```

Comment:
```suggestion
    args += if OS.mac?
      %w[
        --with-apple-sectrust
        --with-gssapi
      ]
    else
      ["--with-gssapi=#{Formula["krb5"].opt_prefix}"]
    end
```
`<<` is not the same as `+=`:
```
irb(main):001> test = []
=> []
irb(main):002> test << [1,2,3]
=> [[1, 2, 3]]
irb(main):003> test += [4,5,6]
=> [[1, 2, 3], 4, 5, 6]
irb(main):004>
```